tenets/codelingo/code-review-comments/context-first-arg/codelingo.yaml

```diff
@@ -0,0 +1,28 @@
+tenets:
+  - name: context-first-arg
+    flows:
+      codelingo/docs:
+        title: Context as First Argument
+        body: |
+          Values of the context.Context type carry security credentials, tracing information, 
+          deadlines, and cancellation signals across API and process boundaries. Go programs 
+          pass Contexts explicitly along the entire function call chain from incoming RPCs 
+          and HTTP requests to outgoing requests.
+
+          Most functions that use a Context should accept it as their first parameter.
+      codelingo/review:
+        comment: Most functions that use a Context should accept it as their first parameter.
+    query: |
+      import codelingo/ast/go
+
+      go.field_list(depth = any):
```

Comment:
field_list could also be the list of return parameters. Unfortunately there is no easy way around this. put a "TODO use macro fact to ensure this is not a return parameter"